### PR TITLE
Add images to folders in Web-UI

### PR DESCRIPTION
**Pull request by KarlStraussberger**
grafik

### CODE REVIEW

File: src/web/containers.cc [hunk 1]
```diff
@@ -63,7 +65,7 @@ void web::containers::process()
 
     auto param = std::make_unique<BrowseParam>(parentID, BROWSE_DIRECT_CHILDREN | BROWSE_CONTAINERS);
     auto arr = database->browse(param);
-
+    std::string virtualUrl = "http://" + Server::getIP() + ":" + Server::getPort() + RequestHandler::joinUrl({ SERVER_VIRTUAL_DIR });
```
[the remark this answers is not in the document]
Comment (KarlStraussberger) [reply src/web/containers.cc:68]: That's why I marked it draft. It's still dirty, but it already works.

[the remark this answers is not in the document]
Comment (whyman) [reply src/web/containers.cc:68]: If I can just rid of the pesky upnp_quirks access then we can bin off getIP() and getPort() entirely.

But I have been meaning to do that for a few _years_ at this point :laughing:

**Comment by whyman** [comment]
Nice, this is actually something I have wanted since I originally forked MediaTomb, so its great to see it coming to life! :tada:

File: src/web/containers.cc (hunk 1 above)
[the remark this answers is not in the document]
Comment (KarlStraussberger) [reply src/web/containers.cc]: It's gone !

**Comment by KarlStraussberger** [comment]
For the release notes: A rescan of the media is necessary to get the container images as resources.

**Comment by whyman** [comment]
Re: art resource rescan, does it impact UPNP clients too?

**Comment by KarlStraussberger** [comment]
> 
> 
> Re: art resource rescan, does it impact UPNP clients too?
grafik

It's not necessary for media. For clients displaying the container image there is still the "old" way which checks for matching files and all media in folder for an image. As soon as I worked out a way to define any container image (maybe after 1.7.0) I'm going to remove the old way which does work on browse instead of scan, i.e. each time instead of once!

**Comment by whyman** [comment]
Nice, anything that involves less work during a client call is worth it.

**Pull request merged**